README.md

```diff
@@ -1,44 +1,110 @@
+# Pets
+
 [![Code Climate](https://codeclimate.com/github/dirtycoder/pets/badges/gpa.svg)](https://codeclimate.com/github/dirtycoder/pets)
 [![Build Status](https://travis-ci.org/dirtycoder/pets.svg?branch=next)](https://travis-ci.org/dirtycoder/pets)
 [![Coverage Status](https://coveralls.io/repos/github/dirtycoder/pets/badge.svg?branch=next)](https://coveralls.io/github/dirtycoder/pets?branch=next)
 [![Requirements Status](https://requires.io/github/dirtycoder/pets/requirements.svg?branch=next)](https://requires.io/github/dirtycoder/pets/requirements/?branch=next)
 
-## Pets
-
-It's a website where people can publish their lost pets,
+Pets is a website where people can publish lost pets
 and pets available for adoption.
 
 Users can create an account with a username and password,
-or they can use Twitter and Facebook to login. You can
-also extend it to use other providers as I use
-python-social-auth here.
+or they can use their Twitter and Facebook to login. You can also extend it to use other providers as it's backed by [python-social-auth](https://pypi.python.org/pypi/python-social-auth).
+
+Images uploaded by users are cropped with [easy-thumbnails](https://pypi.python.org/pypi/easy-thumbnails) to improve the site performance.
+
+**It's still a work in progress**
+
+
+## Installing
 
-Images uploaded by users are cropped with easy-thumbnails
-to improve the site performance.
+### Requirements
 
-It's still a work in progress
+* [Python](https://python.org) 3.4 or newer
+* [PostgreSQL](https://www.postgresql.org) running with a _database_, _username_ and _password_ to be used with Pets.
+
+
+### Fork and clone the repository
+
+First fork the project using GitHub, than clone it locally:
+
+```console
+git clone https://github.com/<username>/pets.git
+cd pets
+```
 
+### Configure your instance
 
-### How to contribute to the project
+The project configuration uses [python-decouple](https://pypi.python.org/pypi/python-decouple/) to dynamically read environment variables and `.env` files.
 
-1. Fork and clone the repository;
-2. Configure your instance (python-decouple);
-   * Remember to configure the username, password and database in your postgreSQL. 
-3. Install Pillow dependencies;
-4. Install PhantonJS, see the julionc [tutorial](https://gist.github.com/julionc/7476620);
-5. Install project requirements;
-6. Execute all tests, it will take some minutes. 
+If you want, you can get startted by copying `contrib/sample-env` as `.env`:
 
 ```console
-git clone https://github.com/<username>/pets.git && cd pets
 cp contrib/sample-env pets/.env
+```
+
+Then you have to set following variables.
+
+#### Basic Django settings
+
+* **`SECRET_KEY`**: [Django's secret key](https://docs.djangoproject.com/en/1.9/ref/settings/#std:setting-SECRET_KEY)
```

Comment:
As the project still uses the LTS version (1.8) what do you think about using the corresponding version of the docs?